tests/faucet.test.ts

```diff
@@ -34,63 +28,74 @@ async function setupAndFundFaucet (t) {
+    entropy.programs.dev.deploy(faucetProgram, configurationSchema, auxDataSchema)
+  )
   t.equal(faucetProgramPointer, LOCAL_PROGRAM_HASH, 'Program pointer matches')
 
-  // register with faucet program
-  await run('Register Faucet Program for eve', accountService.register(
-    { 
+  // Register with faucet program
+  const genesisHash = await entropy.substrate.rpc.chain.getBlockHash(0)
+  const userConfig = {
+    max_transfer_amount: 20_000_000_000,
+    genesis_hash: stripHexPrefix(genesisHash.toString())
+  }
+  await run(
+    'Register Faucet Program for eve',
+    account.register({
       programModAddress: entropy.keyring.accounts.registration.address,
-      programData: [{ program_pointer: faucetProgramPointer, program_config: userConfig }]
-    }
-  ))
-  const verifyingKeys = await faucetService.getAllFaucetVerifyingKeys(entropy.keyring.accounts.registration.address)
+      programData: [{
+        program_pointer:
+        faucetProgramPointer,
+        program_config: userConfig
+      }]
+    })
+  )
+
+  // Fund the Faucet
+  const verifyingKeys = await faucet.getAllFaucetVerifyingKeys(entropy.keyring.accounts.registration.address)
   // @ts-expect-error
-  const { chosenVerifyingKey, faucetAddress } = faucetService.getRandomFaucet([], verifyingKeys)
-  // adding funds to faucet address
-  await run('Transfer funds to faucet address', transferService.transfer(faucetAddress, "1000"))
+  const { chosenVerifyingKey, faucetAddress } = faucet.getRandomFaucet([], verifyingKeys)
+  await run('Transfer funds to faucet address', transfer.transfer(faucetAddress, "1000"))
 
   return { faucetProgramPointer, chosenVerifyingKey, faucetAddress }
 }
 
 test('Faucet Tests: Successfully send funds and register', async t => {
-  const { run, endpoint, entropy: naynayEntropy } = await setupTest(t)
+  const { run, endpoint, entropy: naynay } = await setupTest(t)
+  const { faucetAddress, chosenVerifyingKey, faucetProgramPointer } = await setupAndFundFaucet(t)
 
-  const faucetService = new EntropyFaucet(naynayEntropy, endpoint)
-  const balanceService = new EntropyBalance(naynayEntropy, endpoint)
+  const faucet = new EntropyFaucet(naynay, endpoint)
+  const balance = new EntropyBalance(naynay, endpoint)
 
-  const { faucetAddress, chosenVerifyingKey, faucetProgramPointer } = await setupAndFundFaucet(t)
-  
-  let naynayBalance = await balanceService.getBalance(naynayEntropy.keyring.accounts.registration.address)
+  let naynayBalance = await balance.getBalance(naynay.keyring.accounts.registration.address)
   t.equal(naynayBalance, 0, 'Naynay is broke af')
-  
-  const transferStatus = await run('Sending faucet funds to account', faucetService.sendMoney(
-    {
-      amount: "20000000000",
-      addressToSendTo: naynayEntropy.keyring.accounts.registration.address,
+
+  const amount = 20000000000
+  const transferStatus = await run(
+    'Sending faucet funds to account',
+    faucet.sendMoney({
+      amount: `${20000000000}`,
```

Comment:
```suggestion
      amount: `${amount}`,
```

If the hardcoded number is what is expected here, ignore this